completion.go

```diff
@@ -0,0 +1,34 @@
+package task
+
+import (
+	_ "embed"
+	"fmt"
+)
+
+//go:embed completion/bash/task.bash
+var bash string
```

Comment:
```suggestion
var completionBash string
```

These variables are package wide available. So available via the IDE completion. For this reason, I would expect them to be prefixed.

Reply:
This is a fair request given that they're now in the top-level `task` package. I've updated.